centreon/www/install/steps/process/process_step_vault.php

```diff
@@ -66,7 +66,7 @@
     $encryption->setFirstKey($_ENV["APP_SECRET"]);
     $writeVaultConfigurationRepository =
         new \Core\Security\Vault\Infrastructure\Repository\FsWriteVaultConfigurationRepository(
-            '/var/lib/centreon/vault/vault.yaml',
+            '/var/lib/centreon/vault/vault.json',
```

Comment:
Do not hard-code the library path.
Use the configuration file instead, which already defines it

Reply:
At this step of the install, the configuration file does not exists yet

Reply:
but as I saw i can get this path from install.conf.php 👍